pkg/router/f5/f5.go

```diff
@@ -459,6 +459,35 @@ func (f5 *f5LTM) delete(url string, result interface{}) error {
 }
 
 //
+// iControl REST resource helper methods.
+//
+
+// iControlResourceUriPath returns a fully qualified `escaped` resource URI path component that can be used in the iControl REST calls.
+func (f5 *f5LTM) iControlResourceUriPath(resourceName string) string {
+	resourcePath := path.Join(f5.partitionPath, resourceName)
+	return escapeiControlUriPath(resourcePath)
+}
+
+// iControlVserverUriPath returns a fully qualified `escaped` Virtual Server path that can be used in the iControl REST calls.
+func (f5 *f5LTM) iControlVserverUriPath(vserverName string) string {
+	// Build the fully qualified Virtual Server name.
+	// Note: Although the configuration is stored inside a partition path, the F5 virtual server
+	// belongs to the partition's top-level folder (directory).
+	// Example for a vserver call foo-server, this would be
+	//    /Common/foo-server for the default /Common path
+	// and
+	//    /ose/foo-server for a custom /ose/v3/config path.
+	partitionDir := path.Join("/", F5DefaultPartitionPath)
```

Comment:
Why not just use `partitionDir := F5DefaultPartitionPath`? (If we need this at all—see following comment.)